automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookPage.scala

```diff
@@ -156,10 +168,20 @@ class NotebookPage(override val url: String)(override implicit val authToken: Au
     val jsEscapedCode = StringEscapeUtils.escapeEcmaScript(code)
     executeScript(s"""arguments[0].CodeMirror.setValue("$jsEscapedCode");""", cell)
     clickRunCell(timeout)
-    await condition (cellIsRendered(cellNumber), timeout.toSeconds)
+    await condition(cellIsRendered(cellNumber), timeout.toSeconds)
     cellOutput(cell)
   }
 
+  //TODO: this function is duplicative of the above but does not have the bug
```

Comment:
This function does not wait for the cell to return though -- but I agree it does not have the above bug. Maybe note the difference in a comment.